.babelrc

```diff
@@ -1,12 +1,18 @@
 {
 	"presets": [
-		"env",
-		"react",
-		"stage-0"
+		"@babel/preset-env",
+		"@babel/preset-react",
+		"@babel/preset-stage-0"
```

Comment:
Do we need stage-0 what feature are we using that is not covered by env?

Reply:
I don't know, does env include class properties and spread operators?

Reply:
spread is included in 2015.

Class properties is in stage-2. So I guess this could be "up/down"-graded, to that. but out of the scope for this!